src/AzureAuth/Commands/Ado/CommandToken.cs

```diff
@@ -3,59 +3,127 @@
 
 namespace Microsoft.Authentication.AzureAuth.Commands.Ado
 {
+    using System;
     using System.Collections.Generic;
+    using System.IO;
+
     using McMaster.Extensions.CommandLineUtils;
+
+    using Microsoft.Authentication.AzureAuth.Ado;
     using Microsoft.Authentication.MSALWrapper;
     using Microsoft.Extensions.Logging;
+    using Microsoft.Office.Lasso.Interfaces;
     using Microsoft.Office.Lasso.Telemetry;
 
+    using NLog;
+
     /// <summary>
     /// ADO Command for using either an ADO PAT or acquiring an AAD Access Token.
     /// </summary>
     [Command("token", Description = @"Get a PAT from env vars or an AAD AT for Azure Devops.
 For use by short-lived processes. More info at https://aka.ms/AzureAuth")]
     public class CommandToken
     {
+        private const string OutputOption = "--output";
+        private const string OutputOptionDescription = "How to print the token. One of [token, header, headervalue].\nDefault: token";

```

Comment:
[nit] - This description is confusing to me. It starts with "How to print the token" which seems like you will provide steps to print them. Can we simple say `Token options: [token , header, headervalue].` something?

Reply:
Is this more clear?
```
Options:

  --output        Controls token output format. Options: [token, header, headervalue].
                  Default: token
```

Reply:
yeah works!

Reply:
Great, I will update that in a follow up PR.